pkg/interface/src/logic/api/graph.ts

```diff
@@ -328,14 +328,15 @@ export default class GraphApi extends BaseApi<StoreState> {
 
   async getNewest(ship: string, resource: string, count: number, index = '') {
     const data = await this.scry<any>('graph-store', `/newest/${ship}/${resource}/${count}${index}`);
+    data['graph-update'].fetch = true;
```

Comment:
I assume we currently always use this method to fetch the first page of results in our various graph apps?

Reply:
Yeah, when we aren't loading the entire graph at once.